pyquil/tests/test_quil.py

```diff
@@ -270,7 +270,7 @@ def test_measurement_calls():
 
 def test_measure_all():
     p = Program()
-    p.measure_all((0, MemoryReference('ro', 0)), (1, MemoryReference('ro', 1)), (2, MemoryReference('ro', 3)))
+    p.inst(Declare('ro', 'BIT', 4)).measure_all((0, MemoryReference('ro', 0)), (1, MemoryReference('ro', 1)), (2, MemoryReference('ro', 3)))
```

Comment:
This could be cleaned up with `Program.declare()`:

```
p = Program()
mem = p.declare("ro", memory_size=4)
p.measure_all((0, mem[0]), ...)
```

Reply:
Oh, ok. Thanks, I'll do that!